spec/mail/fields/common/common_field_spec.rb

```diff
@@ -16,15 +17,15 @@
     it "should return '' on to_s if there is no value" do
       expect(Mail::SubjectField.new(nil).to_s).to eq ''
     end
-    
+
     it "should leave ascii alone" do
       field = Mail::SubjectField.new("This is a test")
       expect(field.encoded).to eq "Subject: This is a test\r\n"
       expect(field.decoded).to eq "This is a test"
     end
-    
+
     it "should encode a utf-8 string as utf-8 quoted printable" do
-      value = "かきくけこ"
+      value = String.new("かきくけこ")
```

Comment:
I assumed maybe 1.9 didn't obey the encoding or something -- can't remember off the top of my head what versions supported what.  Since it was in the test already I wasn't interested in removing it and if it can be removed it should probably be in another PR